### ISSUE TITLE
Default timezone not used with fromObject

### ISSUE THREAD

**Issue by ericmorand** [moment/luxon#144]
According to the documentation:

> By default, Luxon creates DateTimes in the system's local zone. However, you can override this behavior globally:

But it doesn't seem to work with DateTime created using fromObject:

```
        DateTimeSettings.defaultZoneName = 'UTC';

        let dt = DateTime.fromObject({year: 2010, month: 10, day: 4, hour: 13, minute: 45});

        console.warn(dt);
```

```
DateTime {
  ts: 2010-10-04T13:45:00.000+02:00,
  zone: Europe/Paris,
  locale: en-US }

```

Although explicitely passing the timezone leads to the correct result:

```
        DateTimeSettings.defaultZoneName = 'UTC';

        let dt = DateTime.fromObject({year: 2010, month: 10, day: 4, hour: 13, minute: 45, zone: 'UTC'});

        console.warn(dt);
```
```
DateTime {
  ts: 2010-10-04T13:45:00.000Z,
  zone: UTC,
  locale: en-US }
```

**Comment by icambron** [comment]
Yeah, that looks like a bug.

**Comment by icambron** [comment]
Fixed in afad8fa, released in 0.2.12.

Thanks for the report!
